src/Modules/StatCan.OrchardCore.EmailTemplates/Controllers/EmailTemplateController.cs

```diff
@@ -315,28 +319,31 @@ public async Task<IActionResult> SendEmail(string name)
 
             if (!templatesDocument.Templates.ContainsKey(name))
             {
-                return RedirectToAction("Index");
+                return Redirect(returnUrl);
             }
 
             var template = templatesDocument.Templates[name];
+            var contentItem = await _contentManager.GetAsync(contentItemId);
```

Comment:
Does this handle both use cases, I think the contentItemId is optional no ?

Reply:
ContentItemId is optional. GetAsync just returns null and the liquid renderer ignores it. From what I tested it works fine.